homepage/index.html

```diff
@@ -0,0 +1,214 @@
+                    <li> Fee Structure</li>
+                    <li> Career Support</li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="layout2">
+            <form action="">
+                <input type="text" id="fullname" name="fullname" placeholder="FULL NAME">
+                <input type="tel" id="phone" name="phone" placeholder="PHONE">
+                <input type="email" id="emailid" name="emailid" placeholder="EMAIL ID">
+                <select name="course" id="course" required>
+                    <option value="" disabled selected hidden>COURSE</option>
+                    <option value="ACCESS 101">ACCESS 101</option>
+                    <option value="ACCESS 102">ACCESS 102</option>
+                    <option value="ACCESS 103">ACCESS 103</option>
+                </select>
+            </form>
+            <button>
+                <p>Send me the Layout</p>
+            </button>
+        </div>
+    </div>
+
+    <div class="online">
+        <div class="flex1">
+            <span>Online or Offline?</span>
+            <p>The only difference between our Online and Offline classes is where you’re located. Both options provide:
+            </p>
+            <ul>
+                <li>Live, Real-Time Instruction</li>
+                <li>Scheduled Lectures and Projects</li>
+                <li>Direct access with Mentors</li>
+                <li>Access Study Material</li>
+                <li>Community Networking</li>
+                <li>Career Support</li>
+            </ul>
+            <div class="online-text">
+                <p>If you’d like to know more—call, email, or chat with us.</p>
+            </div>
+            <button>
+                <p>Talk to Admissions</p>
+            </button>
+        </div>
+        <div class="flex2">
+
+            <img src="/homepage/images/Group 328.png" alt="">
+        </div>
+    </div>
+
+
+    <div class="covid">
+        <p>Due to COVID we are only offereing ONLINE Classes</p>
+    </div>
+
+    <div class="apply">
+        <span>How to Apply?</span>
+        <p>Attend the DEMO Class a week prior of starting of every batch. If satisfied with the approach of the mentors,
+            then Apply for the course. <br>
+            <br>
+            Applying is the first step toward attending access programme and changing or starting your tech career.
+            Learn about the coding bootcamp application process and speak to our admissions counselor.
+        </p>
+        <button>
+            <p>Details</p>
+        </button>
+    </div>
+
+
+    <div class="career-support">
+        <p>Career Support Programme</p>
+        <div class="container">
+            <div class="image-container"><img src="/homepage/images/career support.png" alt=""></div>
+            <div class="text-container">
+                <p>Let’s walk together in your toughest time. After the completion of course, you shall be looking for
```

Comment:
give left and right margin and it should align left

<img width="314" alt="Screenshot 2021-09-17 at 8 43 59 AM" src="https://user-images.githubusercontent.com/47788183/133718418-ebe6c311-140f-4d01-8679-5ebde3dd8412.png">

Reply:
done